### PR TITLE
zlib : Add libMinizip build to package

### PR DESCRIPTION
**Pull request by LongChair**
Thiw will add libminizip.a to sysroot so that some programs can link to it.

### CODE REVIEW

**Comment by lrusak** [comment]
is there no way to have libminizip built as part of the package?

**Comment by LongChair** [comment]
I haven't found any option in zlib to have it build it in zlib build

File: packages/compress/zlib/package.mk [hunk 1]
```diff
@@ -45,3 +55,18 @@ pre_build_host() {
   mkdir -p $PKG_BUILD/.$HOST_NAME
   cp -RP $PKG_BUILD/* $PKG_BUILD/.$HOST_NAME
 }
+
+post_make_target() {
+ # make minizip
```
Comment (stefansaraev) [inline packages/compress/zlib/package.mk]: keep it simple
```
post_make_target() {
  make -C $ROOT/$PKG_BUILD/.$TARGET_NAME/contrib/minizip
}
```

File: packages/compress/zlib/package.mk [hunk 2]
```diff
@@ -45,3 +55,18 @@ pre_build_host() {
...
+ cd $ROOT/$PKG_BUILD/.$TARGET_NAME/contrib/minizip
+ make
+ cd $ROOT/$PKG_BUILD/.$TARGET_NAME/
+}
+
+post_makeinstall_target() {
+ # Install minizip
```
Comment (stefansaraev) [inline packages/compress/zlib/package.mk]:
```
post_makeinstall_target() {
  make -C $ROOT/$PKG_BUILD/.$TARGET_NAME/contrib/minizip DESTDIR=$SYSROOT_PREFIX install
}
```

File: packages/compress/zlib/package.mk [hunk 3]
```diff
@@ -36,6 +36,16 @@ PKG_AUTORECONF="no"
 TARGET_CONFIGURE_OPTS="--prefix=/usr"
 HOST_CONFIGURE_OPTS="--prefix=$ROOT/$TOOLCHAIN"
 
+post_configure_target() {
+ ## configure minizip
+ cd $ROOT/$PKG_BUILD/.$TARGET_NAME/contrib/minizip
+ rm Makefile
+ export PKG_CONFIG_PATH="$PKG_CONFIG_PATH:../../"
+ do_autoreconf
+ ./configure --host=$TARGET_NAME --build=$HOST_NAME $TARGET_CONFIGURE_OPTS
```
Comment (stefansaraev) [inline packages/compress/zlib/package.mk]: --disable-shared --enable-static if you are going to use my post_makeinstall_target below

Comment (stefansaraev) [inline packages/compress/zlib/package.mk]: you can run this all in a subslehll so you dont have to care about $PWD

```
( cd $ROOT/$PKG_BUILD/.$TARGET_NAME/contrib/minizip
  do_autoreconf
  # .. whatever
)
```

**Comment by LongChair** [comment]
@stefansaraev : updated it, thanks for all those comments ;)

**Comment by lrusak** [comment]
good to go?

**Comment by LongChair** [comment]
it think it is :)

**Comment by lrusak** [comment]
thanks!

**Pull request merged**